librz/asm/arch/hexagon/hexagon.h

```diff
@@ -135,6 +138,7 @@ typedef struct {
 	RzList *const_ext_l; // Constant extender values.
 	RzAsm rz_asm; // Copy of RzAsm struct. Holds certain flags of interesed for disassembly formatting.
 	RzConfig *cfg;
+	RzList /* RzAsmTokenPattern */ *token_patterns; ///< NULL terminated arry with token patterns.
```

Comment:
A typo. Also, I wonder if using `RzPVector` better for this - likely you wont need to have random insertion/deletion for those, right?